build.gradle

```diff
@@ -97,25 +98,27 @@ allprojects {
         type = 'IU'
         version = "IU-${ideaVersion}"
         pluginName = "intellij-haxe-${ideaVersion}"
-        ideaDependencyCachePath "${ideaBaseDir}"
+        ideaDependencyCachePath = "${ideaBaseDir}"
         // Specify the sandbox so that Gradle doesn't try to make it relative to each sub-project.
-        sandboxDirectory "${project.rootDir}/build/idea-sandbox"
+        sandboxDir = "${project.rootDir}/build/idea-sandbox"
         // Don't let gradle fill in since/until, we fill them via patchCustomTags using the properties file.
-        updateSinceUntilBuild false
+        updateSinceUntilBuild = false
 
         // Include the "java" built-in plugin for 2019.2 and later builds.
         if (requiresJavaPlugin) {
             System.println("Including java plugin for version >= 2019.2")
-            plugins += 'java'  // 2019.2 + only.  Causes build errors on earlier versions.
+            plugins.add('java') // 2019.2 + only.  Causes build errors on earlier versions.
         }
         // Flex support was unbundled for version 2020.2.
         if (requiresFlexPlugin) {
             System.println("Including Flex plugin for version >= 2020.2")
             String versionWithMajor = ideaVersion.substring(0, 6);
             switch (versionWithMajor) {
-                case "2020.2" : plugins += 'com.intellij.flex:202.6397.59'; break
-                case "2020.3" : plugins += 'com.intellij.flex:203.5981.155'; break
-                case "2021.1" : plugins += 'com.intellij.flex:211.6693.111'; break
+                case "2020.2" : plugins.add('com.intellij.flex:202.6397.59'); break
+                case "2020.3" : plugins.add('com.intellij.flex:203.5981.155'); break
+                case "2021.1" : plugins.add('com.intellij.flex:211.6693.111'); break
+                case "2022.3" : plugins.add('com.intellij.flex:223.7571.4'); break
```

Comment:
tested specifically on `2022.3`, not compatible with `2022.3.3`

Reply:
i think its about time we drop support for older versions  if they are holding us back due to API changes etc. but perhaps a do a release for all older versions before we move on.  it would be nice if we could follow the latest version so people can upgrade their IDEs